README.md

```diff
@@ -10,7 +10,14 @@ This server looks for Bürgeramt appointment every few seconds, and broadcasts t
 pip install -r requirements.txt
 python3 appointments.py
 ```
+### Telegram Notification
 
+Now it is possible to get notified via telegram regarding the appointment dates along with the appointment url. Just click on it and book it right away.
+
+- Go to config directory and rename the config.bak.json to config.json
+
+- Add the telegram api key and the chat id in config.json
```

Comment:
```suggestion
- Add the Telegram API key and the chat id in config.json
```

It might be worth describing how to obtain these too - e.g. https://github.com/inverse/termin#telegram